client/src/components/SponsorsGuests/GuestSection.tsx

```diff
@@ -0,0 +1,84 @@
+// components/GuestSection.tsx
+import { Montserrat } from "next/font/google";
+import Image from "next/image";
+
+const montserrat = Montserrat({
+  subsets: ["latin"],
+  weight: ["400", "600", "700"],
+});
+
+// Type definition for a guest item
+type Guest = {
+  name: string;
+  description: string;
+  link: string;
+  image?: string; // optional photo URL
+};
+
+interface GuestSectionProps {
+  guests: Guest[]; // list of guests passed from parent component
+}
+
+// GuestSection component renders the special guests section
+const GuestSection = ({ guests }: GuestSectionProps) => {
+  return (
+    <div
+      className="mt-12 w-full px-8 py-16"
+      style={{ backgroundColor: "#54595F" }}
```

Comment:
Similar to the fonts, you can also import custom colours into tailwind

https://tailwindcss.com/docs/color#customizing-your-theme

If for whatever reason you can't, you can also style it like this `className="bg-[#54595F]"` instead of having it on a separate line using `style`.